eva_submission/nextflow/validation.nf

```diff
@@ -26,17 +22,15 @@ params.help = null
 if (params.help) exit 0, helpMessage()
 
 // Test input files
-if (!params.vcf_files || !params.reference_fasta || !params.reference_report || !params.output_dir) {
-    if (!params.vcf_files)    log.warn('Provide a input vcf file using --vcf_files')
-    if (!params.reference_fasta)    log.warn('Provide a fasta file for the genome using --reference_fasta')
-    if (!params.reference_report)    log.warn('Provide a assembly report file for the genome using --reference_report')
+if (!params.vcf_files_mapping || !params.output_dir) {
+    if (!params.vcf_files_mapping)    log.warn('Provide a csv file with the mappings (vcf, fasta, assembly report) --vcf_files_mapping')
     if (!params.output_dir)    log.warn('Provide an output directory where the reports will be copied using --output_dir')
     exit 1, helpMessage()
 }
 
 // vcf files are used multiple times
-vcf_channel1 = Channel.fromPath(params.vcf_files)
-vcf_channel2 = Channel.fromPath(params.vcf_files)
+Channel.fromPath(params.vcf_files_mapping).splitCsv(header:true).map{row -> tuple(file(row.vcf), file(row.fasta), file(row.report))}.set{vcf_channel1}
+Channel.fromPath(params.vcf_files_mapping).splitCsv(header:true).map{row -> tuple(file(row.vcf), file(row.fasta), file(row.report))}.set{vcf_channel2}
```

Comment:
This is slick, I might suggest breaking the line up for readability and using [`into`](https://www.nextflow.io/docs/latest/operator.html#into) rather than `set`.  Something like:
```suggestion
Channel.fromPath(params.vcf_files_mapping)
    .splitCsv(header:true)
    .map{row -> tuple(file(row.vcf), file(row.fasta), file(row.report))}
    .into{vcf_channel1; vcf_channel2}
```